srcpkgs/mongoose/template

```diff
@@ -1,29 +1,54 @@
 # Template file for 'mongoose'
 pkgname=mongoose
-version=6.18
-revision=4
+version=7.6
+revision=1
 build_style=gnu-makefile
-make_use_env=compliant
+make_use_env=yes
+make_check_target=test
+make_build_target=linux-libs
 makedepends="openssl-devel"
-short_desc="Easy to use Web server"
-maintainer="Orphaned <orphan@voidlinux.org>"
+checkdepends="openssl-devel"
+short_desc="Embedded Networking Library for TCP,UDP,HTTP,MQTT,WebSocket and SSL/TLS"
+maintainer="Subhaditya Nath <sn03.general@gmail.com>"
 license="GPL-2.0-only"
-homepage="https://cesanta.com/"
+homepage="https://mongoose.ws/"
 distfiles="https://github.com/cesanta/mongoose/archive/${version}.tar.gz"
-checksum=f5c10346abc9c72f7cac7885d853ca064fb09aad57580433941a8fd7a3543769
+checksum=1ef09d971b6de1a6317c109980d6fb5a9c19b39efef2506d6b76869644b3dafa
+CFLAGS="-fPIC -Wno-error=sign-conversion -Wno-error=unused-function"
+# -Wno-error=unused-function	fix-tests.patch
+# -Wno-error=sign-conversion	GCC falsely flags musl FD_{SET,ISSET,CLR}
+#	https://inbox.vuxu.org/musl/20200803174707.GF6949@brightrain.aerifal.cx/T/
+
+
+# used in Makefile
+export SSL=OPENSSL
+
+# libsanitizer-devel is currenly not available on musl
+if [ "$XBPS_TARGET_LIBC" = musl ]
```

Comment:
Make the style of this block consistent with other tests in templates:
- `then` on same line as `if`
- `else` on its own line
- test bodies indented with a single tab